### PR TITLE
Code enhanced - Standardize field name #14697

### PR DESCRIPTION
**Pull request by aspangaro**
Facture & facture_rec

### CODE REVIEW

**Comment by aspangaro** [comment]
Ping @eldy Can you merge this functionnality ? Please

**Comment by eldy** [comment]
Situation really not enough stable for the moment (too much re-engeneering already merged)... Need to wait a stabilization...
I am still making regression test on some external modules to see impact...

File: htdocs/compta/facture/class/facture-rec.class.php [hunk 1]
```diff
@@ -83,11 +83,23 @@ class FactureRec extends CommonInvoice
 	public $number;
 	public $date;
 	public $remise;
+	public $db_table;
```
Comment (eldy) [inline htdocs/compta/facture/class/facture-rec.class.php:86]: What is this property for ?

File: htdocs/compta/facture/class/facture-rec.class.php [hunk 2]
```diff
@@ -83,11 +83,23 @@ class FactureRec extends CommonInvoice
...
+	public $propalid;
```
Comment (eldy) [inline htdocs/compta/facture/class/facture-rec.class.php:87]: What is this property for ?

**Comment by daraelmin** [comment]
Excuse me but why are you renaming fields in french and not in english ? 
Should'nt total_tva and total_ht called total_vat total_excl_tax ?

File: htdocs/compta/facture/class/facture-rec.class.php (hunk 2 above)
Comment (eldy) [reply htdocs/compta/facture/class/facture-rec.class.php]: @aspangaro What is this property for ?

**Pull request merged**